index.js

```diff
@@ -16,19 +16,19 @@ function getDBClient () {
   return client;
 }
 
-// TODO: fetch the user by the user_id
+// fetch the user by the user_id
 async function getUser (client, user_id) {
-  
+  const response = await client.query('SELECT * FROM users WHERE id = $1', [user_id]);
+  return response.rows;
```

Comment:
hey @luanluu168, thanks for making these changes! here we'll want to return `response.rows[0]`. since the calling function is expecting to get back a single user object. i will make this change and merge this branch!

Reply:
Thank you, @karankwatra, for pointing that out as well as merging the pull request.